docs/dev.md

```diff
@@ -74,4 +74,15 @@ Now simply reload the extension. It should connect to React Devtools.
 
 Releases are built automatically and available on the
 [releases pages on Github](https://github.com/mozilla/normandy-devtools/releases).
-There is no manual build process, since it relies on having secrets to sign the add-on.
\ No newline at end of file
+There is no manual build process, since it relies on having secrets to sign the add-on.
+
+### Development releases
+
+We now provide automatically built and signed XPIs for development purposes. These can 
+be found by looking at the list of checks for the commit you want an XPI for and
+click through to the TaskCluster task for "dep-signing-normandy-devtools".
+
+You should now be able to find an XPI file in the list of artifacts.
+
+To be able to install these XPIs you need to go to `about:config` and add a preference
+named `xpinstall.signatures.dev-root` as a boolean set to `true`.
```

Comment:
All my AMO signed extensions got disabled this morning:
<img width="683" alt="image" src="https://user-images.githubusercontent.com/987136/81072999-fc842e80-8eb4-11ea-97c1-eb3f6464df3b.png">

This is fixed by setting `xpinstall.signatures.required` to `false`. Setting this to false removes the need for the other setting. I'm pretty sure this is how I had my old profile set up, but is this a known issue?

Reply:
Yeah, this is the system working as intended. With `xpinstall.signatures.dev-root` set to true, Firefox uses an entirely different root cert to validate add-ons. It would be nice if it would accept both. I think we should explain the problem and advise not using testing builds on real profiles instead of suggesting we disable add-on signature checking.